easybuild/easyblocks/w/wps.py

```diff
@@ -106,6 +106,9 @@ def configure_step(self):
             wrfdir = os.path.join(wrf, det_wrf_subdir(get_software_version('WRF')))
         else:
             raise EasyBuildError("WRF module not loaded?")
+        netcdf_fortran = get_software_root('NETCDFMINFORTRAN')
```

Comment:
From what I have seen recently, the correct flags for this are
```
preconfigopts = "export NETCDF=$EBROOTNETCDFMINFORTRAN && export NETCDF_C=$EBROOTNETCDF &&"
```

Reply:
You would also need to version check these settings (they are only valid for 4.6.something onwards)

Reply:
Oops, WPS, this may be only valid for WRF

Reply:
I removed the change for WRF, but WPS still requires it.